packages/web-react/src/components/FileUploader/FileUploaderAttachment.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { createRef, RefObject, useEffect } from 'react';
+import classNames from 'classnames';
+import { SpiritFileUploaderAttachmentProps } from '../../types';
+import { useStyleProps } from '../../hooks';
+import { useFileUploaderStyleProps } from './useFileUploaderStyleProps';
+import { Icon } from '../Icon';
+
+const FileUploaderAttachment = (props: SpiritFileUploaderAttachmentProps) => {
+  const { label, name, id, onDismiss, buttonLabel = 'Remove', file, ...restProps } = props;
```

Comment:
I think a `buttonLabel` should be required without a default value.

Reply:
Please refer to some points in the code. Make me wrong. We do not want the developer to be surprised by untranslated unexpected defaults. We do not want to deal with translations. Proof me wrong, please.

Reply:
You want examples? 

- HeaderDialogCloseButton (label)
- ModalComposedHeader (closeLabel)
- Tooltip (closeLabel)

We already discussed this topic with Adam, that they have it used like this in RUI. So I created an [issue](https://jira.lmc.cz/browse/DS-765) for it